.rubocop.yml

```diff
@@ -22,3 +22,6 @@ Naming/FileName:
   Enabled: false
 Style/RedundantReturn:
   Enabled: false
+AllCops:
+  Exclude:
+    - 'Boltdir/site-modules/metrics/spec/fixtures/**/*'
```

Comment:
Looks like a new line is needed here.

Reply:
I get the following error when running the `lint:rubocop` task:
```
➜  gatling-puppet-load-test git:(SLV-430) ✗ be rake lint:rubocop
Running RuboCop...
Error: The `Layout/IndentArray` cop has been renamed to `Layout/IndentFirstArrayElement`.
(obsolete configuration found in vendor/bundle/ruby/2.3.0/gems/net-ssh-5.2.0/.rubocop_todo.yml, please update it)
The `Style/MethodMissing` cop has been split into `Style/MethodMissingSuper` and `Style/MissingRespondToMissing`.
(obsolete configuration found in vendor/bundle/ruby/2.3.0/gems/net-ssh-5.2.0/.rubocop_todo.yml, please update it)
RuboCop failed!
```

I recently saw a similar error when I tried adding an exclude to the `.rubocop.yml` file.

Reply:
It looks like it is because I use `bundle install --path vendor/bundle`. I guess that directory is excluded by default but not if you specify an exclusion. I no longer get the error after excluding `vendor`:
```
AllCops:
  Exclude:
    - 'Boltdir/site-modules/metrics/spec/fixtures/**/*'
    - 'vendor/**/*'
```